### PR TITLE
Reverts stomach

### PR DESCRIPTION
**Pull request by jknpj**
Yes...it is true that our stomach will disappear in the course of further evolution..Scientists claim that it  had some use in the past,when early man originated from the apes..But as early man further evolved into mordern spessman this stomach gradually lost its function and started to decrease in size due to its disuse...As Space Homo sapiens sapiens (scientific name of mordern spessman) are evolving further they will eventually loose this stomach (evolution is a continous process)...and humans will be left with even less organs in the future...

Closes #19537
Closes #19478

:cl:
 * rscdel: There' ll come a time when our stomach will disappear.
 * rscdel: (Non-meme version: The stomach organ and all related changes have been reverted.)

### CODE REVIEW

**Comment by bathosbathtime** [comment]
Odd PR name given what is being changed but the commit is accurate so I guess it is good enough.  A good change too as far as I can tell.

**Comment by jknpj** [comment]
Forgot to push the last change before making the PR 🤦‍♂️

**Comment by bathosbathtime** [comment]
Seems drastic but I still support these changes.  The ability to do a chemical purge on someone dead was an interesting niche, but foam, smoke, or some third type of chemical reaction I forgot about can do that so it isn't too big of a loss.

**Comment by Chazach** [comment]
very based

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1633,13 +1633,10 @@ mob/living/carbon/human/isincrit()
...
 	return internal_organs_by_name["appendix"]
 
-/mob/living/carbon/human/get_stomach()
-	return internal_organs_by_name["stomach"]
-
 //Moved from internal organ surgery
 //Removes organ from src, places organ object under user
 //example: H.remove_internal_organ(H,H.internal_organs_by_name["heart"],H.get_organ(LIMB_CHEST))
-/mob/living/carbon/human/remove_internal_organ(var/mob/living/user, var/datum/organ/internal/targetorgan, var/datum/organ/external/affectedarea)
+mob/living/carbon/human/remove_internal_organ(var/mob/living/user, var/datum/organ/internal/targetorgan, var/datum/organ/external/affectedarea)
```
Comment (MadmanMartian) [inline code/modules/mob/living/carbon/human/human.dm]: Der fukken

File: code/modules/mob/mob.dm [hunk 2]
```diff
@@ -1682,9 +1682,6 @@ var/list/slot_equipment_priority = list( \
 /mob/proc/Dizzy(amount)
 	dizziness = max(dizziness,amount,0)
 
-/mob/proc/AdjustDizzy(amount)
```
Comment (MadmanMartian) [inline code/modules/mob/mob.dm]: Der. Fukken.

File: code/modules/organs/organ_internal.dm [hunk 3]
```diff
@@ -229,6 +229,8 @@
 	removed_type = /obj/item/organ/internal/appendix
 
 /datum/organ/internal/proc/remove(var/mob/user, var/quiet=0)
+
+
```
Comment (MadmanMartian) [inline code/modules/organs/organ_internal.dm]: Der. Absoluten. Fukken.

Comment (bathosbathtime) [reply code/modules/organs/organ_internal.dm]: I agree.  It is a shame that the original stomach code PR had these unatomic fixes in them.

**Comment by SonixApache** [comment]
meme description, meme changelog?

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Please merge this, charcoal and holy water are totally broken right now

**Comment by iFlashYou** [comment]
merge when?

**Comment by Exxion** [comment]
A better changelog is important, since this has significant gameplay impact

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
You can, like, edit it before merge

**Comment by Exxion** [comment]
I'd rather give him the opportunity
But yeah, I will if he doesn't tomorrow
Assuming my power isn't out

**Comment by Exxion** [comment]
Opportunity: Given
Power: Flickering incessantly but not out
Commencing operation edit it before merge

**Pull request merged**